libs/ast_generic/AST_generic.ml

```diff
@@ -423,7 +491,7 @@ and id_info = {
   id_info_id : id_info_id; [@equal fun _a _b -> true]
 }
 
-and id_info_id = IdInfoId.t
+and id_info_id = (IdInfoId.t[@name "id_info_id_t"])
```

Comment:
maybe explain what's this @name is, at least the first time we encounter one in the file.